<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>618</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Golos UI</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Параметры</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(195, 255, 195, 255), stop:1 rgba(246, 255, 246, 255));</string>
  </property>
  <widget class="QFrame" name="gridFrame">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>10</y>
     <width>502</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Golos UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <family>Golos UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>154</number>
      </attribute>
      <column>
       <property name="text">
        <string>Размер поля</string>
       </property>
       <property name="font">
        <font>
         <family>Golos UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Сложность</string>
       </property>
       <property name="font">
        <font>
         <family>Golos UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Время</string>
       </property>
       <property name="font">
        <font>
         <family>Golos UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QComboBox" name="comboBox_2">
        <property name="font">
         <font>
          <family>Golos UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Сложность</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Лёгкая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Средняя</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Сложная</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <family>Golos UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Размер поля</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Маленький</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Средний</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Большой</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Golos UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Фильтры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
